context/app/static/js/components/search/Search.tsx

```diff
@@ -179,8 +179,8 @@ function Bar({ type }: TypeProps) {
       </Box>
       <MetadataMenu type={type} />
       <WorkspacesDropdownMenu type={type} />
-      <BulkDownloadButtonFromSearch type={type} />
       {view === 'tile' && <TilesSortSelect />}
+      <BulkDownloadButtonFromSearch type={type} />
```

Comment:
It might also make sense to hide this button in tile view, as you can't select datasets to download - open to ideas on this though.

Reply:
Perhaps a future improvement could be adding selection checkboxes to tiles?